lxd/identities.go

```diff
@@ -2149,10 +2150,20 @@ func updateIdentityCache(d *Daemon) {
 			idpGroupMapping[apiIDPGroup.Name] = apiIDPGroup.Groups
 		}
 
+		var identityIDsWithMultipleSigningKeys []int
+		bearerIdentitySecrets, identityIDsWithMultipleSigningKeys, err = dbCluster.GetAllBearerIdentitySigningKeys(ctx, tx.Tx())
+		if err != nil {
+			return err
+		}
+
+		if len(identityIDsWithMultipleSigningKeys) > 0 {
+			logger.Warn("Found identities with multiple signing keys - these will be omitted from the identity cache", logger.Ctx{"identity_ids": identityIDsWithMultipleSigningKeys})
+		}
+
 		return nil
 	})
 	if err != nil {
-		logger.Warn("Failed reading certificates from global database", logger.Ctx{"err": err})
+		logger.Warn("Failed reading identities from global database", logger.Ctx{"err": err})
```

Comment:
ideally would have been a separate commit so we can backport the fix without the bearer stuff